scripts/cli/updates/1.4.0.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env
+
+source $(pwd)/keg
+
+source $(pwd)/scripts/cli/updates/update_helpers.sh
+
+keg_reset_default_env(){
+  local KEG_DEFS=~/.kegConfig/defaults.env
+  if [[ -f "$KEG_DEFS" ]]; then
+    rm -rf ~/.kegConfig/defaults.env
+  fi
+
+  cp $(pwd)/scripts/setup/defaults.env ~/.kegConfig/defaults.env
+}
+
+keg_cli_1_4_0_update(){
+
+  keg_message "Running update for version 1.4.0..."
+
+  keg_reset_default_env
```

Comment:
This line is not needed, and really shouldn't be called. 
* I had it in a past update because I changed a few things that required removing and re-adding it.
* Moving froward, just calling `keg config sync --confirm false` will sync any new ENVs whith the `global default.env` file